### PR TITLE
odps: add page

### PR DESCRIPTION
**Pull request by onesuper**
odps is the command line tool of a open data processing service.

homepage:
https://www.aliyun.com/product/odps/?lang=en 

yum/apt repo to odpscmd:
http://repo.aliyun.com/odpscmd/

### CODE REVIEW

File: pages/common/odps.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# odps
+
+> Aliyun ODPS command line tool.
```
Comment (waldyrious) [inline pages/common/odps.md]: It would be nice to expand the ODPS acronym here.

File: pages/common/odps.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+- Start the command line:
+
+`odpscmd --config {{odps_config.ini}}`
```
Comment (waldyrious) [inline pages/common/odps.md]: Is there a default config? Ideally we'd start with that, the simplest example, but I see this tool is quite versatile, so we can use --config here, but we should mention it in the command description -- e.g. "Start the command line with a custom configuration file".

**Comment by waldyrious** [comment]
Nice, thanks. but the page is a bit too long. Ideally we'd only showcase the most common 5-6 usages -- which examples do you think could be removed to shorten this page a bit?

**Comment by waldyrious** [comment]
(note that tldr doesn't aim to be comprehensive, just provide a quick overview/reminder)

**Comment by onesuper** [comment]
Yep. This tool is quite complex to use. So I try to write a tl;dr for it. I will reduce the number of example to obey the project's style. : )
But this tool has many concepts and uses a lot of subcommands. Maybe I could add more pages to show the case of sub-commands like git, e.g. odps-resource, odps-table

**Comment by waldyrious** [comment]
The updated page has adequate size. If you feel it doesn't cover the tool in a way that's useful, feel free to create multiple pages for each subcommand. For for now this is a great start. 👍 from me :)

**Comment by onesuper** [comment]
Thanks :-)
Let's merge!

**Comment by felixonmars** [comment]
:+1:

**Pull request merged**